changelogs/fragments/5619-keycloak-improvements.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
```

Aren't these bugfixes and not new features?
```suggestion
bugfixes:
```